birthdays-project/BirthdayApp/models.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+
+
+class Birthdate(models.Model):
+    dob = models.DateField(max_length=8)
```

Comment:
rename it to `date_of_birth`.

`max_length` is not necessary on DateField, so only use it if there's a good reason. Otherwise, other developers spend a lot of time on thinking about WHY it has been set here :) (general remark/strategy)